python/example_code/deeplens/deeplens_inference_lambda.py

```diff
@@ -204,22 +204,15 @@ def infinite_infer_run():
 # All Rights Reserved.                               *
 #                                                    *
 # *****************************************************
-""" A sample lambda for cat-dog detection"""
```

Comment:
This is going to move the imports out of the snippet, which is going to be less readable in the docs.